1-js/02-first-steps/01-hello-world/2-hello-alert-ext/task.md

```diff
@@ -1,9 +1,9 @@
-importance: 5
+Trudność: 5
```

Comment:
In main repo I found this `Please don't translate "importance" (and other top metadata).`

Reply:
thanks